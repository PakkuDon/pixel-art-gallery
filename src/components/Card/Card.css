.card {
  border: 1px solid #333;
  box-shadow: 5px 5px 0 #aaa;
  padding: 1em;
}
