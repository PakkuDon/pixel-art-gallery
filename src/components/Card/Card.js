import React from "react"

import "./Card.css"

const Card = ({ children }) => <div className="card">{children}</div>

export default Card
