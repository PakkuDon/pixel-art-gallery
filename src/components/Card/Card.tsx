import React from "react"

import "./Card.css"

interface CardProps {
    children: React.ReactChildren
}
const Card = ({ children }: CardProps) => <div className="card">{children}</div>

export default Card
