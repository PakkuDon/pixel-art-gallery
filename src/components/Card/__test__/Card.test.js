import React from "react"
import { render } from "@testing-library/react"

import Card from "../Card"

describe("Card", () => {
  it("matches snapshot", () => {
    const children = <div>Hello world</div>
    const { container } = render(<Card>{children}</Card>)

    expect(container).toMatchSnapshot()
  })
})
