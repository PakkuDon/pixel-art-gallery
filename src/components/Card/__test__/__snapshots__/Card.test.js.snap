// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card matches snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <div>
      Hello world
    </div>
  </div>
</div>
`;
